### ISSUE TITLE
docker-compose rm container after docker-compose run

### ISSUE THREAD

**Issue by juliengarcia** [docker/compose#2568]
Hello,

I have a simple docker-compose.yml
```yaml
helloworld:
  build: .
```

I run 

```
docker-compose run helloworld
```

The container run and exit, then I would like to delete this container.
I tried

```
docker-compose rm -f -v helloworld
```

but it's not working:

```
> docker-compose rm -f helloworld
No stopped containers
```

But the container is there:

```
CONTAINER ID        IMAGE               COMMAND                   CREATED             STATUS                      PORTS               NAMES
a7776041eece        sample_helloworld   "/bin/sh -c 'echo \"He"   40 seconds ago      Exited (0) 40 seconds ago                       sample_helloworld_run_1
```

If I run the compose with `docker-compose up`, I can do `docker-compose rm -f -v helloworld`

I know I can do `docker-compose run --rm helloworld` but I would like to know if this behaviour is correct?

Thanks

**Comment by dnephin** [comment]
It is correct. #2184 is an issue to include a way to remove run containers as well

**Comment by juliengarcia** [comment]
Thanks Daniel I'll follow this issue!

Le mar. 22 déc. 2015 17:31, Daniel Nephin <notifications@github.com> a
écrit :

**Comment by simonvanderveldt** [comment]
@dnephin Can we reopen this? #2184 has been closed because the reported switched to #2593 but that seems to cover a different situation (though maybe it's the same code?)

**Comment by dnephin** [comment]
True, there are two related but slightly different issues here. I've opened #2774 to document the problem and proposed solutions.
